moto/guardduty/responses.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import unicode_literals
+from moto.core.responses import BaseResponse
+from .models import guardduty_backends
+import json
+
+
+class GuardDutyResponse(BaseResponse):
+    SERVICE_NAME = "guardduty"
+
+    @property
+    def guardduty_backend(self):
+        return guardduty_backends[self.region]
+
+    # @property
```

Comment:
This commented out code shouldn't be necessary, that's all handled in the parent class (BaseResponse)